service/version.go

```diff
@@ -22,9 +22,9 @@ func (uis *UIServer) versionPage(w http.ResponseWriter, r *http.Request) {
 	project, err := projCtx.GetProject()
 
 	grip.DebugWhen(err != nil || project == nil || projCtx.Version == nil, message.Fields{
-		"message": "error getting project for version page",
-		"project": project,
-		"projCtx": projCtx,
+		"message":            "error getting project for version page",
+		"project_identifier": project.Identifier,
```

Comment:
This can panic if the project is nil. If it's nil, we can probably log an empty string instead.

Reply:
Nice catch!